native/node_addons/TsPreparer/ts_preparer_impl.c

```diff
@@ -592,6 +592,7 @@ prepare_ts_data(
 			// append the ts header
 			if (!append_buffer(output_header, buffers_cur->data, metadata_header.ts_header_size))
 			{
+                printf("\nfail in TS_buffer\n");
```

Comment:
@MosheMaorKaltura this seems great, yet prints should be refined , please advise